models/base_model/arima.py

```diff
@@ -0,0 +1,108 @@
+import logging
```

Comment:
Can you change the name to include "run" in it? So everyone know that this is the run file. Something like: "arima_model_run.py"